src/EditorFeatures/Core/GoToDefinition/IGoToDefinitionService.cs

```diff
@@ -14,10 +12,9 @@ namespace Microsoft.CodeAnalysis.Editor
 {
     internal interface IGoToDefinitionService : ILanguageService
     {
-        /// <summary>
-        /// Finds the definitions for the symbol at the specific position in the document.
-        /// </summary>
-        Task<IEnumerable<INavigableItem>> FindDefinitionsAsync(Document document, int position, CancellationToken cancellationToken);
+        /// <inheritdoc cref="CodeAnalysis.GoToDefinition.IFindDefinitionService.FindDefinitionsAsync(Document, int, CancellationToken)"/>
+        // Keep changes to this method in sync with CodeAnalysis.GoToDefinition.IFindDefinitionService
+        Task<IEnumerable<INavigableItem>?> FindDefinitionsAsync(Document document, int position, CancellationToken cancellationToken);
```

Comment:
Feel free to also add an Obsolete attribute here asking people to move to IFindDefinitionService.  We can then work with partners to get them moved over, and can then remove this.

Reply:
I'm assuming we don't want to do this until we migrate TS and F#.